README.md

```diff
@@ -85,6 +85,23 @@ You can now build Terra Core. Running the following command will install executa
 make install
 ```
 
+Add `GOROOT` and `GOPATH` to your environment `PATH`
```

Comment:
I think this section should go to `Step 1. Install Golang`. Also, isn't this a part of Golang documentation? What if they change it? I am afraid of making broken dependency.

Reply:
I agree, let's close this